compare-heights.js

```diff
@@ -0,0 +1,363 @@
+// types
+const { ApiPromise } = require('@polkadot/api');
+const util = require('util');
+const request = require("request");
+const requestPromise = util.promisify(request);
+
+//Import typeReg-istry
+const getTypeRegistry = require('@polkadot/types/codec/typeRegistry').default;
+
+const { Enum, EnumType, Struct, Option, Vector } = require ('@polkadot/types/codec');
+
+// const { getTypeRegistry, u32, u64, Bool, Text, BlockNumber, Moment, AccountId, Hash, Balance, Bytes } = require ('@polkadot/types');
+const { u32, u64, Bool, Text, BlockNumber, Moment, AccountId, Hash, Balance, Bytes } = require ('@polkadot/types');
+const { u8aToString } = require('@polkadot/util');
+
+
+// media-types
```

Comment:
All the joystream runtime types are now nicely packaged up in `@joystream/types` npm package so we replace all the types definition and registrations here with:

```
const { registerJoystreamTypes }  = require('@joystream/types');
registerJoystreamTypes();
```